src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -309,7 +313,14 @@ def build_bundle(self) -> ArtifactDeploymentMap:
         """
         Populates the local deploy root from artifact sources.
         """
-        return build_bundle(self.project_root, self.deploy_root, self.artifacts)
+        mapped_files = build_bundle(self.project_root, self.deploy_root, self.artifacts)
+        if FeatureFlag.ENABLE_SETUP_SCRIPT_GENERATION.is_enabled():
+            _find_and_execute_processors(
```

Comment:
remind me: where does the output of this step go?

Reply:
We do not have the generate DDL piece in yet, which I can direct to stdout. 
If you just want to see the JSON output for now, I can output that instead. Will add the change in the next commit.